src/components/POLine/Item/ErrorMessage.test.js

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@folio/jest-config-stripes/testing-library/react';
+
+import { ErrorMessage } from './ErrorMessage';
+
+const renderErrorMessage = (props = {}) => render(
+  <ErrorMessage>{props}</ErrorMessage>,
```

Comment:
The approach to rendering `props` objects seems confusing to me. If `props` is a React element, it'll work, but in the other, it will break.